src/lib/transition_frontier/persistent_frontier/rpc_worker.ml.ignore

```diff
@@ -1,30 +1,22 @@
 open Core
```

Comment:
what's the deal with this rename?

Reply:
The persistent_frontier library was removed entirely. I renamed it locally initially to keep the files around, but I guess that I didn't set up my .gitignore right and they got added to history. Removing.